src/backends/backend/python/resources/triton_python_backend_utils.py

```diff
@@ -226,6 +257,90 @@ def message(self):
         -------
         str
             The message associated with this exception, or None if no message.
-
         """
         return self._msg
+
+
+def get_input_tensor_by_name(inference_request, name):
+    """Find an input Tensor in the inference_request that has the given
+    name
+
+    Parameters
+    ----------
+    inference_request : InferenceRequest
+        InferenceRequest object
+    name : str
+        name of the input Tensor object
+
+    Returns
+    -------
+    Tensor
+        The input Tensor with the specified name, or None if no
+        input Tensor with this name exists
+    """
+    input_tensors = inference_request.inputs()
+    for input_tensor in input_tensors:
+        if input_tensor.name() == name:
+            return input_tensor
+
+    return None
+
+
+def get_input_properties_by_name(model_config, name):
```

Comment:
Why is it get_input_properties... and get_output_config...?
Use config for both.